core/src/test/java/org/elasticsearch/index/IndexWithShadowReplicasIT.java

```diff
@@ -80,6 +82,11 @@ private Settings nodeSettings(String dataPath) {
                 .build();
     }
 
+    @Override
+    protected Collection<Class<? extends Plugin>> nodePlugins() {
+        return pluginList(MockTransportService.TestPlugin.class);
```

Comment:
Hmm.. I see the downside for this being that with this the plugin is used suite-wide instead of for the single test as before?